### PR TITLE
Do not add padding in carusel block

### LINKED ISSUE #56910: Rebalance theme: Post Carousel block display broken if "Pages" are selected
### Quick summary

If you set up a Post Carousel block on the Rebalance theme, and allow it to display Pages (alone or with other content types), the block displays well in the editor, the preview, and on smaller devices. However, on larger devices, the featured images appear shifted.

### Steps to reproduce

1. Activate the `Rebalance` theme on your site.
2. On a new page, add the `Post Carousel` block. (Note: This bug is **not** present on posts.)
3. In the block settings panel, under "Post Types," select `Pages`.
4. To ensure a page displays, use the "Choose Specific Posts" setting to specify at least one page on your site.
5. Save changes.

### What you expected to happen

I expected the Post Carousel to display on the live site exactly as it does in the editor:
Markup 2021-10-08 at 13 16 12

### What actually happened

On larger screens *only*, it instead displays with the featured image shifted:

https://user-images.githubusercontent.com/5630717/136597165-baa0a45c-1ae5-408f-a3ce-5fa253d1b44d.mp4

### Context

4359830-zen
User had a Post Carousel block set to display a handful of specific pages on their site. It works perfectly on mobile devices, but not on a desktop/laptop.

### Operating System

_No response_

### Browser

_No response_

### Simple, Atomic or both?

Simple, Atomic

### Theme-specific issue?

Specific to Rebalance, as far as I can tell. Tested on Twenty Twenty One, and couldn't replicate it there.

### Other notes

_No response_

### Reproducibility

Consistent

### Severity

Some (< 50%)

### Available workarounds?

No but the platform is still usable

### Workaround details

_No response_

### PR DESCRIPTION
**Pull request by gikaragia**
Fixes: https://github.com/Automattic/wp-calypso/issues/56910

#### Changes proposed in this Pull Request:
In wp.com, there is an issue with the carousel block on pages, where the pages appear broken, see linked issue. This PR fixes this issue

#### Related issue(s):
Raised initially in: https://github.com/Automattic/wp-calypso/issues/56910

#### How to reproduce and test:
- In an installation that has the [carousel block](https://github.com/Automattic/newspack-blocks/releases) installed, activate the Rebalance theme.
- Create a couple of pages and add a featured image to them.
- On another page add the carousel block
- Observe that the pages appear broken.
- Switch to this branch and try again.
- Observe that the carousel is correct.

#### Before
Screenshot 2024-09-11 at 14 16 02

#### After
Screenshot 2024-09-11 at 14 36 49

### CODE REVIEW

**Comment by gikaragia** [comment]
Hello @MaggieCabrera, I would appreciate a review in this change. Sorry for the ping if I am wrong, not sure what is the correct process to merge and release this one.

**Comment by obenland** [comment]
@gikaragia For anything that changes visually, it's helpful to add before/after screenshots to the PR description to make it easier to see what to look for.

@ianstewart I assume someone from Marvel should review this since this is considered theme maintenance?

**Comment by ianstewart** [comment]
@obenland yep (for themes launched before Jan 2023 like this one) @dsas @alaczek .

**Review by dsas — APPROVED** [review]
Thanks @gikaragia LGTM

The steps to deploy are in the [contributing guide](https://github.com/Automattic/themes/blob/trunk/CONTRIBUTING.md#workflow). You might not have access to do successfully deploy. If not you can just merge this PR and it'll get deployed the next time someone runs a deploy, typically within a few days. Or you can ping me after it is merged and I can deploy it.

**Comment by gikaragia** [comment]
Hey @dsas I tried following the instructions but I am not sure I understand the process. In my sandbox:
- Not sure how I can pull trunk to deploy
- When I look in `package.json` in my sandbox,  I don't have a `deploy` command (neither I have the `theme-utils.mjs` script) so it looks like the whole directory is out of sync

I'm probably doing something totally wrong so I'll leave this to be deployed whenever a regular deployment happens.

**Comment by dsas** [comment]
@gikaragia I've deployed it 👍 

The `npm run deploy` command is ran from your local checkout, it takes care of updating your sandbox etc and deploying.

**Pull request merged**
